content/news/XXXX-XX-XX-russians-war.md

```diff
@@ -0,0 +1,19 @@
+title: Russia's invasion of Ukraine
+authors: Daniel Schürmann
+tags: russian ukraine war invasion
+status: draft
+comments: yes
+
+Since 2022-02-24 we receive horrible news from Ukraine, which have become even worse since then. As an organization committed to freedom we feel the urge speak up and show solidarity for Ukraine.
```

Comment:
```suggestion
On 2022-02-24 we received horrible news from Ukraine, which has become even worse since then. As an organization committed to freedom we feel the urge speak up and show solidarity with Ukraine.
```